client/src/components/ReviewTile.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const ReviewTile = props => {
+  return (
+    <div>
+        <li><p>{props.rating}/5 stars - {props.review}</p></li>
+    </div>
```

Comment:
this div is not doing anything, but we have more going on

the p tag is also not doing anything. Lets just make this one li tag!

Reply:
Done